### ISSUE TITLE
Force-attribute for 2+ attributes for HTML in .liquidrc

### ISSUE THREAD

**Issue by james0r** [panoply/vscode-liquid#89]
Seems like many conventions drop attributes to a new line for 2 or 3+ attributes.

Would be nice to see either only multiple attribute tags dropped to new lines, or have a prop in .liquidrc for `force-attribute-threshhold` or something, after which attributes are forced to new line.

Thanks!

**Comment by panoply** [comment]
Use wrap at `85`, eg:

```
 "html": {
    "correct": true,
    "force_attribute": false,
    "braces": false,
    "preserve": 1,
    "wrap": 85,
    "preserve_text": true,
    "content": true,
    "space_close": true,
    "tags": [
      {
        "tag": "source",
        "rules": {
          "force_attribute": true,
          "space_close": true
        }
      }
    ]
  },
```

**Comment by panoply** [comment]
FYI: If you have Liquid tags you can do this: 

```js
  <nav
    class="accordion mt-3 mb-2"
    data-accordion-collapsible-value=" {{- some_tag }}" // notice the space between quotation and " {{-
    data-accordion-multiselect-value=" {{- some_tag }}"  // notice the space between quotation and " {{-
    data-controller="accordion">
  </nav>
```

It will help prevent the tags from breaking.

**Comment by panoply** [comment]
🚢  Shipped [v3.0.0](https://github.com/panoply/vscode-liquid/releases/tag/v3.0.0)
